From 04ccd0b086181cb31c8a9400fc4c8db5223f8e4e Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Tue, 3 Feb 2015 17:58:20 -0500
Subject: [PATCH 8/8] aufs: restore f_dentry macro

Upstream commit 78d28e651f97866d608d9b41f8ad291e65d47dd5
("kill f_dentry macro") removed the macro, but AUFS was relying
on it quite extensively.  Restore it locally to AUFS.

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 fs/aufs/aufs.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/fs/aufs/aufs.h b/fs/aufs/aufs.h
index d82e9743f4dd..997bd2b9c703 100644
--- a/fs/aufs/aufs.h
+++ b/fs/aufs/aufs.h
@@ -55,5 +55,7 @@
 #include "whout.h"
 #include "wkq.h"
 
+#define f_dentry       f_path.dentry
+
 #endif /* __KERNEL__ */
 #endif /* __AUFS_H__ */
-- 
2.1.0

